configurations/manager/debian10.yaml

```diff
@@ -1,4 +1,4 @@
 ami_id_monitor: 'ami-07d02ee1eeb0c996c'  # Debian buster image - debian-10-amd64-20210208-542
 ami_monitor_user: 'admin'
 
-scylla_repo_m: "http://downloads.scylladb.com.s3.amazonaws.com/deb/ubuntu/scylla-2021.1.list"
+manager_scylla_backend_branch: '2021'
```

Comment:
will it make sense to create a single yaml to have these default in one place?
like creating a new file - "manager_defaults.yaml" with these values of:
manager_scylla_backend_branch: '2020'
manager_branch: 'master_latest'

then you can add "manager_defaults.yaml" to the list of yamls in pipeline file.

Reply:
It's already a part of defaults/test_default.yaml.
If you're talking about manager tests specifically, we can't at the moment, since some of the tests use 2020 and some use 2021